client/packages/android/app/src/main/java/org/openmsupply/client/certplugin/CertPlugin.java

```diff
@@ -74,6 +75,11 @@ public CertWebViewClient(Bridge bridge, File filesDir) {
     @Override
     public void onReceivedSslError(WebView view, SslErrorHandler handler,
                                    SslError error) {
+        // TODO store and trust previous cert
+        if (!error.getUrl().contains("localhost")) {
```

Comment:
technically this wouldn't be checked (since we connect to local IP, not localhost), i'll address this later

Reply:
It's in here: https://github.com/openmsupply/open-msupply/issues/743, soonish (i'll seperate in another issue and add to the milestone), I already spent a lot of time making this work and didn't do it in the PR. 

I still don't understand why checking certs like this is required (vs remembering and comparing next connection and perhaps some validation code/QR at the first connection). Having a malicious server on a tablet device would mean the device is severely compromised, probably rooted at which point you can access app data of any app and read any cert.